hw-model/src/lib.rs

```diff
@@ -975,7 +979,7 @@ mod tests {
             ..Default::default()
         })
         .unwrap();
-        model.step_until_output("hi").unwrap();
+        model.step_until_output("hii").unwrap();
```

Comment:
NP: why did we add an extra "i" ? :)

Reply:
Because sending the same character twice is now a special case (if the toggle bit doesn't change, the second character will be missed), and I wanted to just add a dummy check that it still works in this test.